docs: Add guidelines and update docs for published library from UI (#256)
excalidraw/excalidraw-libraries: README.md

- Open the library menu (by clicking on the "library" button or typing `9`)
- The selected object should appear in a square. Click on it (you will see a `+` sign when hovering over it)

> 4. Still in the library menu, select the libraries you want to publish (Hover over the library items and you should see a checkbox, clicking on which will select the library item).

Reviewer: ```suggestion
4. Still in the library menu, select the libraries you want to publish (hover over a library item and you should see a checkbox, clicking on which will select the library item).
```

> 4. Still in the library menu, select the libraries you want to publish (Hover over the library items and you should see a checkbox, clicking on which will select the library item).

Reviewer: ```suggestion
4. Still in the library menu, select the libraries you want to publish (hover over a library item and you should see a checkbox, clicking on which will select the library item).
```

> 5. Hit the publish button.

Reviewer: ```suggestion
5. Hit the publish button (in blue), and fill out the details.
```

<!--Add a GIF here once the library changes look good!-->

If you want to test your library file, use the "Load" button in the library menu to reimport it.

## Guidelines

> -  All library items should be generic and reusable.

Reviewer: ```suggestion
- All library items should be reusable. Not something that only you or your friend will use.
```

Author: > Not something that only you or your friend will use.
Probably need to reword this.
How about ?
```
All library items should be reusable and not just limited to your usecase
```

Reviewer: Yeah, I updated the change

Author: ok somehow I am not getting the option to commit it so I will push it

Author: ```suggestion
- All library items should be reusable and not just limited to your personal use case.

- Don't republish (copy/paste) items from other libraries without making any significant changes.

> - Each item should stand on its own and add value.
> Eg Adding single square or a squiggly line doesn't meet standard.

Reviewer: ```suggestion
- In general, each item should be usable and stand on its own.
- Don't publish items which are very easy to create directly within the editor. For example, a single arrow or a square.
```

> Check out the published [libraries](https://libraries.excalidraw.com/?theme=light&sort=default) to get started.

Reviewer: ```suggestion
Check out our existing [libraries](https://libraries.excalidraw.com/?theme=light&sort=default) to get started.
```

- Open the library menu (by clicking on the "library" button or typing `9`)
- The selected object should appear in a square. Click on it (you will see a `+` sign when hovering over it)

4. Still in the library menu, select the libraries you want to publish (hover over a library item and you should see a checkbox, clicking on which will select the library item).

5. Hit the publish button (in blue), and fill out the details.

<!--Add a GIF here once the library changes look good!-->

If you want to test your library file, use the "Load" button in the library menu to reimport it.

## Guidelines

- All library items should be reusable and not just limited to your personal use case.
- Don't republish (copy/paste) items from other libraries without making any significant changes.
- In general, each item should be usable and stand on its own.
- Don't publish items which are easy to create directly. For example, a single arrow or a square.

> - Don't publish items which are easy to create directly. For example, a single arrow or a square.

Author: ```suggestion
```
